src/main/resources/templates/db_scripts/schema.json

```diff
@@ -215,6 +215,11 @@
       "fromModuleVersion": "mod-organizations-storage-4.7.0",
       "withOptimisticLocking": "failOnConflict",
       "withMetadata": true,
+      "index": [
+        {
+          "fieldName": "bankAccountNumber"
```

Comment:
I am no sure is it enough just common btree index here, because customer can want to search for example by * or like that bankNumb* and in this case we should apply full text search index.
Lets ask today about expectations

Reply:
Currently search query pattern looks like `${sIndex}="${query}*"` in organization app.

E.g.
![image](https://github.com/folio-org/mod-organizations-storage/assets/88109087/9ca7018f-6d27-4146-a673-aa88b5661c73)

Reply:
@azizbekxm so we should use gin or full text search index

Reply:
If I don't mistake, "=" invoke Full Text search so that, we should use FT index

Reply:
![image](https://github.com/folio-org/mod-organizations-storage/assets/113523904/4901a9c1-45c9-4958-938d-8b0807d943c4)
![image](https://github.com/folio-org/mod-organizations-storage/assets/113523904/0bc9d837-6e41-4228-b7c6-3c35c684107f)